src/terra-dev-site/contributing/ComponentStandards.e.contributing.md

```diff
@@ -0,0 +1,100 @@
+| --------- | --------- |
+| last 2 versions | iOS >= 10 |
+
+A list of the exact browsers this currently includes can be [found here](http://browserl.ist/?q=iOS+%3E%3D+10%2C+last+2+and_chr+versions%2C+last+2+android+versions%2C+last+2+chrome+versions%2C+last+2+edge+versions%2C+last+2+firefox+versions%2C+last+2+ie+versions%2C+last+2+safari+versions).
+
+## Responsive Support
+
+Components should work responsively without introducing horizontal scrollbars on resize.
+
+Terra components are built following the mobile-first approach and utilize the following media query breakpoints:
+
+ - Default: 0+
+ - Tiny: 544px+
+ - Small: 768px+
+ - Medium: 992px+
+ - Large: 1216px+
+ - Huge: 1440px+
+
+## Mobile Support
+
+Components should:
+* Support gestures when possible (swiping / pinching)
+* Not use hover events
+* Avoid small tap target sizes
+
+## Accessibility (A11y)
+
+We should strive to make components accessible. Use the following guides to help build accessible components.
+* [WCAG 2.0](https://www.w3.org/TR/WCAG20/)
+* [Section 508](https://www.access-board.gov/guidelines-and-standards/communications-and-it/about-the-ict-refresh/final-rule/text-of-the-standards-and-guidelines)
+* [Color Contrast Checker](http://webaim.org/resources/contrastchecker/)
+
+Components should support keyboard navigation.
+
+All functionality that is available by mouse or touch should also be available by keyboard. Keyboard operation is possible by moving to ('focusing') interactive elements and activating them using keystrokes.
+
+The important keys for testing are:
+
+* `Tab` for moving focus to the next interactive element
+* `Shift` + `Tab` for moving focus to the previous interactive element
+* `Enter` and/or `Space` for activating (i.e. pressing/clicking) the interactive element
+* `←` and/or `→` for moving focus within tabs and some other custom controls
+* `Esc` for closing menus, dialogs, and popups
+
+Test with screen readers:
+
+* [VoiceOver](https://help.apple.com/voiceover/info/guide/10.12/) on Mac / Safari
+* [Narrator on Windows](https://support.microsoft.com/en-us/help/17173/windows-10-hear-text-read-aloud)
+
+## CSS
+
+Components should follow [https://sass-guidelin.es/](https://sass-guidelin.es/) and our [CSS conventions](#/contributing/terra-ui/conventions) for Terra UI.
+
+## JavaScript
+
+Components should follow [Airbnb javascript style guide](https://github.com/airbnb/javascript) and our [JS conventions](#/contributing/terra-ui/conventions) for Terra UI.
+
+## Testing
+
+Components should include [Jest](https://github.com/facebook/jest) tests and [Webdriver.io](http://webdriver.io/) tests and follow our [testing conventions](#/contributing/terra-ui/conventions) for Terra UI.
+
+## Internationalization (I18n) Support
+
+Components should be internationalized and support the following locales:
+
+* `ar`
```

Comment:
Can we list the langs with the locales like we do in core's wiki? https://github.com/cerner/terra-core/wiki/Component-Features#localization-support Most people off-handedly might not know a locale.

(side note) can we remove core's wiki once this is in?!